third_party/opa/data-and-policies/user-created-policy-3/sample_policies.rego

```diff
@@ -0,0 +1,76 @@
+package dataapi.authz
+
+verdict[output] {
+	count(rule) == 0
+	output = {"action": {"name":"DenyAccess"}, "policy": "Deny by default"}
```

Comment:
Are we going to get DenyAccess instead of Deny? If so, the manager code should be updated accordingly.

Reply:
I can change it to Deny. @shlomitk1 
Will there be any issue in the action returned in that case @rohithdv ?

Reply:
I think this is an important question. We have to now take the action names from the taxonomy definitions. But it may need some discussion.

Reply:
Okay sure. @roee88 Can you please take a look and suggest.

Reply:
@rohithdv @ritwikc Whatever is defined in the taxonomy is fine. I just ask to keep the manager code consistent.

Reply:
Done